src/util/wrap-text.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2024  Puter Technologies Inc.
+ *
+ * This file is part of Phoenix Shell.
+ *
+ * Phoenix Shell is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+// TODO: Detect ANSI escape sequences in the text and treat them as 0 width?
```

Comment:
Add a `// TODO` for multi-byte UTF-8 as well
```suggestion
// TODO: Detect ANSI escape sequences in the text and treat them as 0 width?
// TODO: Ensure this works with multi-byte characters (UTF-8)
```